.github/workflows/release.yaml

```diff
@@ -85,13 +90,27 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GH_LERNA_TOKEN }} # Temporary fix to make lerna able to push the new versions commit to master
           GITHUB_TOKEN: ${{ secrets.GH_LERNA_TOKEN }} # Temporary fix to make lerna able to push the new versions commit to master
-        run: yarn lerna version --conventional-commits --conventional-prerelease --preid rc --create-release github --no-private --yes
-
+        run: |
+          if [[ "${{ github.ref }}" == "ref/head/master" ]]; then
+            if [[ "${{ contains(github.event.commits[0].message, '[bump-version]') }}" == "true"]]; then
+              yarn lerna version --conventional-commits --create-release github --no-private --yes
+            fi
+          elif [[ "${{ github.ref }}" == ref/head/release-*  ]]; then
+            if [[ "${{ contains(github.event.commits[0].message, '[bump-version]') }}" == "true"]]; then
```

Comment:
What does this to, is this something we want?

Reply:
This checks whether the PR from the release-n branch and if the commit message has the tag named [bump-version] then it increases the version of the release candidate

Reply:
The purpose is to keep a constant version in a release branch (for rc) by combining it with [release] tag in the publish logic.